### ISSUE TITLE
Release notes from release candidaets not copied through the official release.

### ISSUE THREAD

**Issue by rjtormis** [release-it/release-it#1127]
![f3c28b4a-4619-4e66-af69-4cf91a67cdf9](https://github.com/release-it/release-it/assets/62842207/faefd5ce-b9fb-46fe-a227-739ae7617797)

Greetings, first of all I appreciate this plugin which really eases up creating and building releases. My question is why are pre-release notes not copied through the official release as seen on the picture.  I have established CICD for this and here is the current set up 

`name: Release candidate

on:
  push:
    branches:
      - testing-branch
jobs:
  release-candidate:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout Code
        uses: actions/checkout@v4
        with:
          fetch-depth: 0

      - name: Setup pnpm
        uses: pnpm/action-setup@v4
        with:
          version: 8

      - name: Setup Node.js
        uses: actions/setup-node@v3
        with:
          node-version: "21.x"
          cache: "pnpm"
          cache-dependency-path: pnpm-lock.yaml

      - name: git config
        run: |
          git config user.name "${GITHUB_ACTOR}"
          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"

      - name: Install dependencies
        run: |
          pnpm install --ignore-scripts
          pnpm build
        env:
          DATABASE_URL_RLS_USER: ${{ secrets.DATABASE_URL_RLS_USER }}
          DATABASE_URL_ROOT_USER: ${{ secrets.DATABASE_URL_ROOT_USER }}

      - name: Release
        run: pnpm release-candidate
        env:
          GITHUB_TOKEN: ${{secrets.GH_TOKEN}}
`

`name: Official Release

on:
  push:
    branches:
      - prod-release

jobs:
  release-official:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout Code
        uses: actions/checkout@v4

      - name: Setup pnpm
        uses: pnpm/action-setup@v4
        with:
          version: 8

      - name: Setup Node.js
        uses: actions/setup-node@v3
        with:
          node-version: "21.x"
          cache: "pnpm"
          cache-dependency-path: pnpm-lock.yaml

      - name: Install dependencies
        run: |
          pnpm install --ignore-scripts

      - name: git config
        run: |
          git config user.name "${GITHUB_ACTOR}"
          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"

      - name: Release
        run: pnpm release
        env:
          GITHUB_TOKEN: ${{secrets.GH_TOKEN}}
`

Basically the flow or what I want to achieve is 

Pull latest ver to the feature branch -> Create pull request to main (staging) -> Once approved create a release candidate -> Push staging to the production and release the official build. 

Hence I wanted to know why pre release notes are not appended on the official release notes. Inputs are appreciated, thanks!

**Comment by webpro** [comment]
That looks a bit odd indeed. Draft releases should have the notes handled the same.

I've tried https://github.com/release-it/release-it/blob/main/docs/github-releases.md#update-the-latest-release a few times (a while ago) and this worked well. In general I don't do draft releases with release-it though (but pre-releases).

**Comment by webpro** [comment]
Btw the config you're showing doesn't show anything specific to release-it?

**Comment by webpro** [comment]
Closing this issue or pull request as part of general housekeeping, to keep this project sustainable and optimize my time working on it. If you think this is inappropriate or if there is no workaround and you feel stuck, feel free to [open a new issue](./new/choose) or [contact me](https://github.com/webpro). Thanks for your understanding.
